### PR TITLE
feat: rename ccd states (CMC-1238) 2/2

### PR DESCRIPTION
**Pull request by pd-dudkiewicz**
To be merged after https://github.com/hmcts/unspec-service/pull/698

Note: Revert sonar scan exclusions: https://github.com/hmcts/unspec-service/pull/698/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R228

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1238

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by swalker125 — COMMENTED** [review]
Should the supression for test coverage (added in 1/2) be removed in this PR?

**Comment by pd-dudkiewicz** [comment]
@swalker125 yeah, will do it once the 1st one gets merged, don't want to base this PR off it to avoid the mess with rebasing

**Review by douglasrice — DISMISSED** [review]

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -234,5 +234,12 @@
         "CRUD": "CRU"
       }
     ]
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "DISMISS_CLAIM",
+    "UserRoles": ["caseworker-civil-solicitor","caseworker-civil-admin"],
+    "CRUD": "CRU"
```
Comment (swalker125) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json]: what's this for?

**Review by swalker125 — DISMISSED** [review]
LGTM, just one comment on event permissions for case dismissed

**Review by douglasrice — APPROVED** [review]

**Review by swalker125 — APPROVED** [review]

**Pull request merged**